src/client/utils/tracking/tracking.ts

```diff
@@ -152,15 +152,14 @@ export const getTrackableContractCategory = (
 }
 
 export const getInitialOfferFromSessionStorage = (
-  contractType: TrackableContractType,
+  contractCategory: TrackableContractCategory,
 ) => {
   const initialtOffer = sessionStorage.getItem('initial_offer')
 
   if (initialtOffer) {
     return initialtOffer
   }
 
-  const contractCategory = getTrackableContractCategory(contractType)
   sessionStorage.setItem('initial_offer', contractCategory)
   return contractCategory
 }
```

Comment:
Wait, you pass in `contractCategory` and then return it at the end of the function? 😅
Seems like you could just skip passing it in/returning it altogether.

I would also break it into two functions; `getInitialOfferFromSessionStorage` + `updateInitialOfferInSessionStorage` for example to avoid causing side effects and getting things in the same function.

Reply:
True you are, a bit lazy of me 😄